### ISSUE TITLE
Gooier Limb Crushing

### ISSUE THREAD

**Issue by SirDoggoTheBoyo** [studio-minus/ppg-bugs-and-requests#1734]
Small Lines just like the Limb attachment ones when a limb is dismembered, but instead, they appear when a body part is crushed
and they go from the nearest object, to whatever object crushed the body part, making a more gorey effect.

I request this feature because the current limb crushing feels a little weak (if thats a good way to describe it?)

<sub>Original on the left, Crappy hashing of what i mean by the description on the right.</sub>
![unknown](https://user-images.githubusercontent.com/60833059/178950193-ef8de0f7-d2b3-4e56-bb44-da4655442922.png)

**Comment by SirDoggoTheBoyo** [comment]
this is a repost due to lack of proper formatting (my bad!)

**Comment by PossiblyRemember** [comment]
This feature would go really well with stringy tissue, I hope this gets added

**Comment by SirAJ4895** [comment]
We NEED this feature in the game!

**Comment by PossiblyRemember** [comment]
> We NEED this feature in the game!

Could ya please stick to the thumbs up?

**Comment by DindinYT37** [comment]
> this is a repost due to lack of proper formatting (my bad!)

You seem to know that editing exists (since you edited this post like 20 times), so why did you repost?

**Comment by SirDoggoTheBoyo** [comment]
> > this is a repost due to lack of proper formatting (my bad!)
> 
> You seem to know that editing exists and how to use it perfectly fine (since you edited this post like 20 times), so why did you repost? Not meant to be rude or anything by the way, just curious.

i also just generally messed up the formatting and i didnt actually classify it as a request, so it was just a loose post sitting around, shouldve explained a little better, my b

**Comment by DindinYT37** [comment]
> > > this is a repost due to lack of proper formatting (my bad!)
> > 
> > 
> > You seem to know that editing exists and how to use it perfectly fine (since you edited this post like 20 times), so why did you repost? Not meant to be rude or anything by the way, just curious.
> 
> i didnt actually label it as a request, so it was just a loose post sitting around, shouldve explained a little better, my b

Oh, I didn't see. Alright then.

**Comment by PossiblyRemember** [comment]
> > > We NEED this feature in the game!
> > 
> > Could ya please stick to the thumbs up?
> 
> I'm sorry but I just like to show extra support. I will stop if it annoys you.

Thanks, thumbs up is usually good enough support :)

**Comment by SirDoggoTheBoyo** [comment]
welp, this thread is completely dead

**Comment by DindinYT37** [comment]
> welp, this thread is completely dead, so i doubt its ever gonna get accepted into the game

Month-old requests have already been accepted, so I wouldn't say this.

**Comment by SirDoggoTheBoyo** [comment]
wait requests can actually be excepted?
thought they were just for concepts

**Comment by PossiblyRemember** [comment]
> wait requests can actually be Accepted? (correction due to stupid phone) thought they were just for concepts

yes, they can be accepted, they can also be rejected, and some people make mods for suggestions.

**Comment by SirDoggoTheBoyo** [comment]
and the threads dead permanantely lol

**Comment by PossiblyRemember** [comment]
> and the threads dead permanantely lol

idk why some of the best requests like #1322 and #1337 (to list a few), among **many** others end up unread by zooi and yet minor issues or requests are the ones noticed

**Comment by mestiez** [comment]
I read these lmao it's just that requests like these take a lot of time to implement so I let them evolve on the board for a while

**Comment by PossiblyRemember** [comment]
> I read these lmao it's just that requests like these take a lot of time to implement so I let them evolve on the board for a while

ah ok, makes sense

**Comment by SirDoggoTheBoyo** [comment]
since i now know zooi atleast reads these, what are the chances of this ever getting added? (if i may ask)

**Comment by mestiez** [comment]
i dont know but i like the idea
